pkg/assets/builder_test.go

```diff
@@ -17,35 +17,42 @@ limitations under the License.
 package assets
 
 import (
-	"k8s.io/kops/pkg/apis/kops"
 	"testing"
+
+	"k8s.io/kops/pkg/apis/kops"
 )
 
 func TestRemap_File(t *testing.T) {
+	t.Skip("not going to run")
```

Comment:
Why are we skipping this?

Reply:
I need to have an interface, so that stuff does not actually d/l.  I am planning to add an interface to the find hash function to mock this out. This code d/l's the file.  I will add a note